BinaryObjectScanner/Protection/CopyX.cs

```diff
@@ -0,0 +1,164 @@
+        // These comments assume with 0 evidence that the former is Light and the latter is Professional.
+        // Because there is no evidence, only copy-X is being returned for now. This check has pre-emptively separated the two,
+        // just for when a designation can be applied for sure.
+
+        // Overall:
+        // Whenever these comments state "at the end of" or "at the start of" pertaining to the filesystem, they refer to alphabetical order, because this is how copy-X images seem to be mastered.
+        // Both Light and Professional have a directory at the end of the image. The files within this directory are intersected by the physical ring.
+        // This file is usually called ZDAT, but not always. At least one instance of Light calls it ZDATA. At least one instance of Professional calls it System.
+        // Seemingly it can be anything. It doesn't help that most known samples are specifically from one company's games, Tivola. Still, most use ZDAT.
+
+        // Professional:
+        // All instances of professional contain a disc check, performed via optgraph.dll. 
+        // All instances of professional contain in the directory at the end of the image 3 files. gov_[something].x64, iofile.x64, and sound.x64.
+        // Due to gov's minor name variance, sound.x64 sometimes being intersected by a ring at the start, and iofile.x64 being referenced directly in optgraph.x64, only iofile.x64 is being checked for now.
+        // TODO: optgraph.dll also contains DRM to prevent kernel debugger SoftICE from being used, via a process called SoftICE-Test. It is not currently known if this is specifically part of copy-X, or if it's an external solution employed by both copy-X and also other companies. If it's the latter, it should have its own check. It has none here since it wouldn't be necessary.
+
+
+        // Light:
+        // All instances of light contain 1 or more files in the directory at the end of the image. They all consist of either 0x00, or some data that matches between entries (and also is present in the 3 Professional files), except for the parts with the rings running through them.
+        // TODO: Check the last directory alphabetically and not just ZDAT*
+
+        /// <inheritdoc/>
+        public string? CheckPortableExecutable(string file, PortableExecutable pex, bool includeDebug)//Checks for Professional
+        {
+            if (pex.OverlayStrings != null)
+            {
+                // Checks if main executable contains reference to optgraph.dll. Emergency 4's is missing this for some reason. 
+                // This might be better removed later, as Redump ID 82475 is a false positive, and also doesn't actually contain the actual optgraph.dll file.
+                // TODO: Find a way to check for situations like Redump ID 48393/, where the string is spaced out with 0x00 between letters and does not show up on string checks.
```

Comment:
Please remove the stray `/` after the Redump ID.

Reply:
Should be fixed